From e516bf9c95a59bfc62098d1e50b5c83accff4f28 Mon Sep 17 00:00:00 2001
From: Joey <joey@lineageos.org>
Date: Sat, 8 Dec 2018 10:55:43 +0100
Subject: [PATCH] overlays: accents: tune for contrast

Tune some colors for improved contrast on both light and dark backgrounds

Signed-off-by: Joey <joey@lineageos.org>
Change-Id: Ida8d0d2622ec6384fe3a2328d4b1a179f482f106
---
 accents/brown/res/values/colors.xml  | 14 +++++++-------
 accents/green/res/values/colors.xml  | 14 +++++++-------
 accents/pink/res/values/colors.xml   | 14 +++++++-------
 accents/red/res/values/colors.xml    | 14 +++++++-------
 accents/yellow/res/values/colors.xml | 14 +++++++-------
 5 files changed, 35 insertions(+), 35 deletions(-)

diff --git a/accents/brown/res/values/colors.xml b/accents/brown/res/values/colors.xml
index 2c22615..9a6e855 100644
--- a/accents/brown/res/values/colors.xml
+++ b/accents/brown/res/values/colors.xml
@@ -15,11 +15,11 @@
     limitations under the License.
 -->
 <resources>
-    <color name="accent_device_default_700">#795548</color>
-    <color name="accent_device_default_dark">#795548</color>
-    <color name="accent_device_default_light">#795548</color>
-    <color name="material_deep_teal_200">#795548</color>
-    <color name="material_deep_teal_500">#795548</color>
-    <color name="notification_icon_bg_color">#795548</color>
-    <color name="system_notification_accent_color">#795548</color>
+    <color name="accent_device_default_700">#996A5A</color>
+    <color name="accent_device_default_dark">#996A5A</color>
+    <color name="accent_device_default_light">#996A5A</color>
+    <color name="material_deep_teal_200">#996A5A</color>
+    <color name="material_deep_teal_500">#996A5A</color>
+    <color name="notification_icon_bg_color">#996A5A</color>
+    <color name="system_notification_accent_color">#996A5A</color>
 </resources>
diff --git a/accents/green/res/values/colors.xml b/accents/green/res/values/colors.xml
index 4a2abd6..c8e4800 100644
--- a/accents/green/res/values/colors.xml
+++ b/accents/green/res/values/colors.xml
@@ -15,11 +15,11 @@
     limitations under the License.
 -->
 <resources>
-    <color name="accent_device_default_700">#4CAF50</color>
-    <color name="accent_device_default_dark">#4CAF50</color>
-    <color name="accent_device_default_light">#4CAF50</color>
-    <color name="material_deep_teal_200">#4CAF50</color>
-    <color name="material_deep_teal_500">#4CAF50</color>
-    <color name="notification_icon_bg_color">#4CAF50</color>
-    <color name="system_notification_accent_color">#4CAF50</color>
+    <color name="accent_device_default_700">#38853B</color>
+    <color name="accent_device_default_dark">#38853B</color>
+    <color name="accent_device_default_light">#38853B</color>
+    <color name="material_deep_teal_200">#38853B</color>
+    <color name="material_deep_teal_500">#38853B</color>
+    <color name="notification_icon_bg_color">#38853B</color>
+    <color name="system_notification_accent_color">#38853B</color>
 </resources>
diff --git a/accents/pink/res/values/colors.xml b/accents/pink/res/values/colors.xml
index 5506aaf..0bb1c33 100644
--- a/accents/pink/res/values/colors.xml
+++ b/accents/pink/res/values/colors.xml
@@ -15,11 +15,11 @@
     limitations under the License.
 -->
 <resources>
-    <color name="accent_device_default_700">#E91E63</color>
-    <color name="accent_device_default_dark">#E91E63</color>
-    <color name="accent_device_default_light">#E91E63</color>
-    <color name="material_deep_teal_200">#E91E63</color>
-    <color name="material_deep_teal_500">#E91E63</color>
-    <color name="notification_icon_bg_color">#E91E63</color>
-    <color name="system_notification_accent_color">#E91E63</color>
+    <color name="accent_device_default_700">#E6125E</color>
+    <color name="accent_device_default_dark">#E6125E</color>
+    <color name="accent_device_default_light">#E6125E</color>
+    <color name="material_deep_teal_200">#E6125E</color>
+    <color name="material_deep_teal_500">#E6125E</color>
+    <color name="notification_icon_bg_color">#E6125E</color>
+    <color name="system_notification_accent_color">#E6125E</color>
 </resources>
diff --git a/accents/red/res/values/colors.xml b/accents/red/res/values/colors.xml
index 356feb3..60a1496 100644
--- a/accents/red/res/values/colors.xml
+++ b/accents/red/res/values/colors.xml
@@ -15,11 +15,11 @@
     limitations under the License.
 -->
 <resources>
-    <color name="accent_device_default_700">#F44336</color>
-    <color name="accent_device_default_dark">#F44336</color>
-    <color name="accent_device_default_light">#F44336</color>
-    <color name="material_deep_teal_200">#F44336</color>
-    <color name="material_deep_teal_500">#F44336</color>
-    <color name="notification_icon_bg_color">#F44336</color>
-    <color name="system_notification_accent_color">#F44336</color>
+    <color name="accent_device_default_700">#D0422D</color>
+    <color name="accent_device_default_dark">#D0422D</color>
+    <color name="accent_device_default_light">#D0422D</color>
+    <color name="material_deep_teal_200">#D0422D</color>
+    <color name="material_deep_teal_500">#D0422D</color>
+    <color name="notification_icon_bg_color">#D0422D</color>
+    <color name="system_notification_accent_color">#D0422D</color>
 </resources>
diff --git a/accents/yellow/res/values/colors.xml b/accents/yellow/res/values/colors.xml
index 71c80a4..dfdb0e4 100644
--- a/accents/yellow/res/values/colors.xml
+++ b/accents/yellow/res/values/colors.xml
@@ -15,11 +15,11 @@
     limitations under the License.
 -->
 <resources>
-    <color name="accent_device_default_700">#FFC107</color>
-    <color name="accent_device_default_dark">#FFC107</color>
-    <color name="accent_device_default_light">#FFC107</color>
-    <color name="material_deep_teal_200">#FFC107</color>
-    <color name="material_deep_teal_500">#FFC107</color>
-    <color name="notification_icon_bg_color">#FFC107</color>
-    <color name="system_notification_accent_color">#FFC107</color>
+    <color name="accent_device_default_700">#F4AC45</color>
+    <color name="accent_device_default_dark">#F4AC45</color>
+    <color name="accent_device_default_light">#F4AC45</color>
+    <color name="material_deep_teal_200">#F4AC45</color>
+    <color name="material_deep_teal_500">#F4AC45</color>
+    <color name="notification_icon_bg_color">#F4AC45</color>
+    <color name="system_notification_accent_color">#F4AC45</color>
 </resources>
-- 
2.17.1

